cmd/deploy.go

```diff
@@ -28,19 +29,16 @@ var deployCommand = &cobra.Command{
 	Short: "[Cluster] Deploy Workspace on cluster if no arguments passed assuming that Workspace is current directory.",
 	Args:  cobra.ExactArgs(1),
 	Run: func(cmd *cobra.Command, apps []string) {
```

Comment:
I want to simplify this:
every method returns error, i want to stop execution and show error. But in current code `err` variable is overwrited and in the end it's show last Error occurred.